errors with nightly zig to "build --release" on windows

I get the following 2 errors when trying to build any of the release type (fast, small, safe) with zig-master.
I tried with:
- zig-windows-x86_64-0.14.0-dev.1457
- zig-windows-x86_64-0.14.0-dev.1417
```
C:\Users\user\AppData\Local\zig\p\12203578c4326bd2088018f5d0f5ad395b6f4ea8228ceec99b60274c4329e832aa5c\src\civetweb\civetweb.c:714:9: error: 'MUST_IMPLEMENT_CLOCK_GETTIME' macro redefined
#define MUST_IMPLEMENT_CLOCK_GETTIME
        ^
<command line>:9:9: note: previous definition is here
C:\Users\user\AppData\Local\zig\p\12203578c4326bd2088018f5d0f5ad395b6f4ea8228ceec99b60274c4329e832aa5c\src\civetweb\civetweb.c:22173:20: error: expansion of date or time macro is not reproducible
  const char *bd = __DATE__;
```
can be reproduced with:
```
zig build run_minimal --release=safe
```

There is no problem with zig-0.13 to "zig build --release" on windows, nor is there any problem with zig-master to just "zig build".
The issue only occurs when using zig-master to "build --release" on windows.

--- a/build.zig
+++ b/build.zig
@@ -102,7 +102,7 @@ pub const V0_12 = struct {
         });
 
         // basical flags for civetweb
-        const basic_flags = [_][]const u8{ "-DNDEBUG", "-DNO_CACHING", "-DNO_CGI", "-DUSE_WEBSOCKET" };
+        const basic_flags = [_][]const u8{ "-DNDEBUG", "-DNO_CACHING", "-DNO_CGI", "-DUSE_WEBSOCKET", "-Wno-error=date-time" };
 
         // when enable tls
         const tls_flags = [_][]const u8{ "-DWEBUI_TLS", "-DNO_SSL_DL", "-DOPENSSL_API_1_1" };
@@ -117,9 +117,6 @@ pub const V0_12 = struct {
         try civetweb_flags.appendSlice(&ubsan_flags);
         try civetweb_flags.appendSlice(&basic_flags);
         try civetweb_flags.appendSlice(if (enable_tls) &tls_flags else &no_tls_flags);
-        if (target.result.os.tag == .windows) {
-            try civetweb_flags.append("-DMUST_IMPLEMENT_CLOCK_GETTIME");
-        }
 
         var webui_flags = std.ArrayList([]const u8).init(b.allocator);
         defer webui_flags.deinit();

